packages/kbn-shared-ux-components/src/solution_avatar/solution_avatar.tsx

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+import './solution_avatar.scss';
+
+import React from 'react';
+
+import { DistributiveOmit, EuiAvatar, EuiAvatarProps } from '@elastic/eui';
+import classNames from 'classnames';
+
+export type KibanaSolutionAvatarProps = DistributiveOmit<EuiAvatarProps, 'size'> & {
+  /**
+   * Any EuiAvatar size available, or `xxl` for custom large, brand-focused version
+   */
+  size?: EuiAvatarProps['size'] | 'xxl';
```

Comment:
Is there any way we can utilize home's `FeatureCatalogueSolution` type here to require a Solution to pass to the `name` and `iconType` props of EuiAvatar? I really like the idea of strongly typing this usage to only be allowed specifically for solutions.

Something like:

```suggestion
  size?: EuiAvatarProps['size'] | 'xxl';
  solution: FeatureCatalogueSolution;
```

Then:

```tsx
<EuiAvatar
  ...
  name={solution.name}
  iconType={solution.icon}
/>
```

Reply:
I like the idea, but unfortunately, that interface is only exported from the plugin, and we can't be adding dependencies on plugins here. But we have a tracking issue for that, so can add there.